src/views/Namespace/Edit.vue

```diff
@@ -14,14 +14,25 @@
         class="mr-1"
       >
         <b-button
-          data-test-id="button-visit-namespace"
+          data-test-id="button-visit-admin panel"
```

Comment:
If you're going to change the cypress id, please, notify Bojan so he can update the needed tests.